apiary.apib

```diff
@@ -8,10 +8,9 @@ The HTTP API is currently accepting all methods, so GET will work just as well a
 
 # Group add [POST /add{?arg}{&r,p,t,n,w,H,s}]
 
-Add a file to IPFS.
+Add a file or directory to IPFS.
 
-Note that directories are added recursively, to form the ipfs
-MerkleDAG.
+Adds contents of <path> to ipfs. Use -r to add directories (recursively).
```

Comment:
`-r` is not really sth you can use on the http api, it's more `r` or `recursive`.